Testscripts/Windows/PERF-APACHE-BENCHMARK.ps1

```diff
@@ -0,0 +1,215 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Those 2x PS1 scripts look alike. Can you combine it?

Reply:
Fixed.